app/server/api/transfer_card_api.py

```diff
@@ -135,13 +139,17 @@ def put(self, public_serial_number, nfc_serial_number):
             }
             return make_response(jsonify(response_object)), 404
 
-        if not transfer_card.transfer_account:
-            response_object = {
-                'message': f'Card {transfer_card.id} not bound to transfer account',
-            }
-            return make_response(jsonify(response_object)), 400
+        if disable:
```

Comment:
Should we `400` if we try to disable the same card twice? Right now it's not allowed on the FE, but you can hit this API over and over

Reply:
done :)